apps/total-typescript/src/pages/welcome/index.tsx

```diff
@@ -201,8 +201,10 @@ const GetStarted: React.FC<React.PropsWithChildren<unknown>> = () => {
   )
 }
 
-const Share: React.FC<React.PropsWithChildren<unknown>> = () => {
-  const tweet = `https://twitter.com/intent/tweet/?text=Just purchased Engineering Management for the Rest of Us by @sarah_edo`
+const Share: React.FC<React.PropsWithChildren<{productName: string}>> = ({
+  productName,
+}) => {
+  const tweet = `https://twitter.com/intent/tweet/?text=Just purchased ${productName} by @${process.env.NEXT_PUBLIC_PARTNER_TWITTER}`
```

Comment:
This bit of Eng Mgmt got through, update it and make it configurable based on the name of the purchased product.